templates/graph-theory.md

```diff
@@ -12,10 +12,149 @@ keywords:
   - topological sort
 ---
 
+
+
+### Depth First Traversal
+
+Go to [Tutorial](../tutorials/graph-theory/binary-tree#traversal)
```

Comment:
Just a **one-liner intro would be good about DFS.** 

And firmly say, **Please refer the tutorial guide for more details.**